functions/HttpTriggerAPIUsersId/function.json

```diff
@@ -8,7 +8,9 @@
       "name": "req",
       "methods": [
         "get",
-        "post"
+        "post",
+        "put",
```

Comment:
POST and PUT methods are defined - Looks good!

Reply:
Are you  planning to implement the PATCH in addition to PUT per Toddy's comments?

Reply:
I will add this to the functions.json file but patch wont be allowed since we follow the lenny's swagger doc